<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>product_one_db</class>
 <widget class="QDialog" name="product_one_db">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>1091</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 127);</string>
  </property>
  <widget class="QLabel" name="label_debug_checker">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 241, 241);</string>
   </property>
   <property name="text">
    <string>&lt;b&gt;This is a text&lt;/b&gt;</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(248, 248, 248);
font-size : 200px;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;
  &lt;head/&gt;
    &lt;body&gt;
        &lt;i&gt;&lt;p&gt;
          &lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;DATABASE Of PRODUCT
          &lt;/span&gt;
        &lt;/p&gt;&lt;/i&gt;
    &lt;/body&gt;
&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>ITEM LIST</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>Grocery Product</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>Fresh Produce</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>Beverage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>Fish Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>Bakery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>XYZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>ABC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>CBD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>NMH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 18px;</string>
    </property>
    <property name="text">
     <string>LKJ</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 255);
font-weight : bold;
font-size : 20px;</string>
   </property>
   <property name="text">
    <string>Billing</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>431</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>70</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Bill</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size : 17px;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 248, 212);
border: none;
color : black;
font-weight : bold;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size : 17px;</string>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_price">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 248, 212);
border: none;
color : black;
font-weight : bold;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size : 17px;</string>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_quantity">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color : white;
font-weight : bold;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addBill">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);
font-weight : bold;</string>
    </property>
    <property name="text">
     <string>Add To Bill</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_code">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>69</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color : white;
font-weight : bold;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size : 17px;</string>
    </property>
    <property name="text">
     <string>Product Code</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>660</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 15px;
</string>
   </property>
   <property name="text">
    <string>Bill</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_logout">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>660</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-weight : bold;
font-size : 15px;</string>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>MyLabell</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_payment">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>660</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 213, 213);
color : black;
font-weight : bold;
font-size : 15px;
</string>
   </property>
   <property name="text">
    <string>Payment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
